packages/dd-trace/test/appsec/waf/waf_context_wrapper.spec.js

```diff
@@ -27,14 +41,45 @@ describe('WAFContextWrapper', () => {
     wafContextWrapper.run(payload)
 
     expect(ddwafContext.run).to.have.been.calledOnceWithExactly(payload, 1000)
+    expect(reportMetricsStub).to.have.been.calledOnce
   })
 
-  it('Should send ephemeral addresses every time', () => {
+  it('Should send HTTP_INCOMING_QUERY twice if waf run returns null', () => {
```

Comment:
not sure what this test is trying to do with the twice thing ? why not call it just once ?